src/TwigExtension.php

```diff
@@ -22,4 +24,94 @@ public function is_instanceof($value, string $type): bool
       || (\function_exists($func = 'is_'.$type) && $func($value))
       || $value instanceof $type;
   }
-}
\ No newline at end of file
+
+  /**
+   * Returns entity ids of entities with matching title/name/label.
+   *
+   * Supported entity types are node, taxonomy_term, group, and user.
```

Comment:
This may go under "entity_type" as a comment

Reply:
Done.